pandas/core/frame.py

```diff
@@ -233,14 +233,13 @@
 copy : bool, default True
     If False, avoid copy if possible.
 indicator : bool or str, default False
-    If True, adds a column to output DataFrame called "_merge" with
-    information on the source of each row.
-    If string, column with information on source of each row will be added to
-    output DataFrame, and column will be named value of string.
-    Information column is Categorical-type and takes on a value of "left_only"
-    for observations whose merge key only appears in 'left' DataFrame,
-    "right_only" for observations whose merge key only appears in 'right'
-    DataFrame, and "both" if the observation's merge key is found in both.
+    If True, adds a column to the output DataFrame called "_merge" with
+    information on the source of each row. The column can be given a different
+    name by providing a string argument. The column will have a Categorical
+    type with the value of "left_only" for observations whose merge key only
+    appears in 'left' DataFrame, "right_only" for observations whose merge key
```

Comment:
'left' -> the left, 'right' -> the right might sound / look better

Reply:
Just adjusted the wording. Let me know if that looks good, or if I should add anything else.

Reply:
I meant replacing `'left'` with `the left` and similarly for `'right'` (i.e., "whose merge key only appears in the left DataFrame...")

Reply:
Sorry, I misunderstood. It should be all set now.